### PR TITLE
UISegmentedControl, UISwitch

### PR DESCRIPTION
**Pull request by gionoa**

### CODE REVIEW

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift [hunk 1]
```diff
@@ -1,19 +1,120 @@
-/*
- - UISegmentedControl
-     - How do you customize it.
-     - How do you add an action/target.
- - UISwitch (Gio)
-     - How do you customize it.
-     - How do you add an action/target.
- */
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift]: Don’t delete this comment please

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift [hunk 2]
```diff
@@ -1,19 +1,120 @@
...
+//            NSAttributedString.Key.font : font,
+//            NSAttributedString.Key.foregroundColor: UIColor.darkGray
+//            ], for: .selected)
+//
+//        segmentedControl.tintColor = .clear
 
+        //segmentedControl.backgroundColor = .gray
+        
+        //segmentedControl.insertSegment(withTitle: "Donald Trump", at: segmentedControl.numberOfSegments + 1, animated: true)
+        //segmentedControl.setTitle(String(segmentedControlItems[0].reversed()), forSegmentAt: 0) /* segment must already exist */
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift:42]: Is this dead code or to demo during the meetup?

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift [hunk 3]
```diff
@@ -1,19 +1,120 @@
...
+            segmentedControl.leadingAnchor.constraint(greaterThanOrEqualTo: view.leadingAnchor),
+            segmentedControl.trailingAnchor.constraint(lessThanOrEqualTo: view.trailingAnchor),
+            textLabel.topAnchor.constraint(equalTo: segmentedControl.bottomAnchor, constant: inset),
+            textLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            textLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: inset),
+            textLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -inset)
+            ])
+    }
+    
+    //MARK: - UIEvent Actions
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift]:
```suggestion
    // MARK: - UIEvent Actions
```

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift [hunk 4]
```diff
@@ -1,19 +1,120 @@
...
+    @objc func segmentValueChanged(_ sender: UISegmentedControl) {
+        textLabel.text = quotes[segmentedControl.selectedSegmentIndex]
+    }
+    
+
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift]: Delete extra space

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift [hunk 5]
```diff
@@ -1,19 +1,120 @@
...
+        view.addConstraints([
+            segmentedControl.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: inset),
+            segmentedControl.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            segmentedControl.heightAnchor.constraint(equalToConstant: 30),
+            segmentedControl.leadingAnchor.constraint(greaterThanOrEqualTo: view.leadingAnchor),
+            segmentedControl.trailingAnchor.constraint(lessThanOrEqualTo: view.trailingAnchor),
+            textLabel.topAnchor.constraint(equalTo: segmentedControl.bottomAnchor, constant: inset),
+            textLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            textLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: inset),
+            textLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -inset)
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift:93]: Nice! You're a pro

**Review by ivancr — CHANGES_REQUESTED** [review]
Very nice work man! Your code is improving so much by the day. Keep it up!

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift (hunk 2 above)
Comment (gionoa) [reply BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift:42]: It's going to be used to demo during the meetup

**Pull request merged**